utils/check_import_finished.php

```diff
@@ -133,7 +133,8 @@ function isReverseOnlyInstallation()
         $print_fail();
         echo <<< END
         Hints:
-        * Rerun the setup.php --create-search-indices step
+        * Rerun the './utils/setup.php --create-search-indices' step
+          without or possibly with '--ignore-errors' parameter.
```

Comment:
I'd simplify things and just say: rerun './utils/setup.php --create-search-indices --ignore-errors'

The `--ignore-errors` is needed to skip over indexes that already exist and at this point the chance are high that at least some of them have already been created.

Reply:
great, even easier then